server/src/resolvers/User.js

```diff
@@ -1,7 +1,7 @@
 const User = {
-  posts: (parent, args, context) => {
-    return context.prisma.user({ id: parent.id }).posts()
-  },
+  children: (parent, args, context) => {
+    return context.prisma.user({ id: parent.id }).children()
```

Comment:
rather than parent.id maybe change this to sitter.id, since the people inputting the data will not actually be the parents of the children but the people watching them